docs/guides/ai-transpiler-passes.mdx

```diff
@@ -102,6 +112,38 @@ The following custom collection passes for Cliffords, Linear Functions and Permu
 
 These custom collection passes limit the sizes of the collected sub-circuits so they are supported by the AI-powered synthesis passes. Therefore, it is recommended to use them after the routing passes and before the synthesis passes for a better overall optimization.
 
+## Hybrid Qiskit-AI circuit transpilation
```

Comment:
```suggestion
## Hybrid heuristic-AI circuit transpilation
```

I think calling it Qiskit might be a bit confusing since we use "Qiskit" also as a brand in addition to the SDK itself.